heroku/test_heroku_2fa.py

```diff
@@ -0,0 +1,65 @@
+# TODO: discover why/how the exemptions.py & regressions.py processess screw up
+#       normal pytest operations.  All the fixtures below could/should be in
+#       the heroku/conftest.py file, and no mark.parametrize would be needed on
+#       the actual tests.
```

Comment:
I think this is due to https://github.com/mozilla-services/pytest-services/blob/master/conftest.py#L174-L196 - We utilize what `parametrize` injects to add additional metadata to the results.

Reply:
Yep, we should fail more gracefully for unparametrized tests.